src/server/qgsservercachefilter.h

```diff
@@ -0,0 +1,136 @@
+/***************************************************************************
+                          qgsservercachefilter.h
+                          ------------------------
+ Cache interface for Qgis Server plugins
+
+  begin                : 2018-07-05
+  copyright            : (C) 2018 by René-Luc D'Hont
+  email                : rldhont at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSSERVERCACHEPLUGIN_H
+#define QGSSERVERCACHEPLUGIN_H
```

Comment:
Can you update these preprocessor macro to be in sync with the current class name?

Reply:
It's like in `src/server/accesscontrolfilter.h` you'll find the preprocessor macro `QGSACCESSCONTROLPLUGIN_H` with `class SERVER_EXPORT QgsAccessControlFilter`

And in `src/server/accesscontrol.h`. You will find the definition of `class QgsAccessControlPlugin;`.